doi-transformer/src/test/java/no/unit/nva/doi/transformer/CrossRefConverterTest.java

```diff
@@ -132,25 +137,29 @@ public void anEmptyCrossRefDocumentThrowsIllegalArgumentException() throws Illeg
     }
 
     @Test
-    @DisplayName("An CrossRef document with unknown type throws IllegalArgument exception")
-    public void anCrossRefDocumentWithUnknownTypeThrowsIllegalArgumentException() throws IllegalArgumentException {
+    @DisplayName("An CrossRef document with unknown type logs error and continues")
+    public void anCrossRefDocumentWithUnknownTypeLogsErrorAndContinues()
+            throws IllegalArgumentException, InvalidIsbnException,
+            UnsupportedDocumentTypeException, InvalidIssnException {
         CrossRefDocument crossRefDocument = sampleCrossRefDocumentWithBasicMetadata();
         crossRefDocument.setType(SOME_STRANGE_CROSSREF_TYPE);
-        UnsupportedDocumentTypeException exception =
-                assertThrows(UnsupportedDocumentTypeException.class, () -> toPublication(crossRefDocument));
+        TestAppender testAppender = LogUtils.getTestingAppender(CrossRefConverter.class);
+        toPublication(crossRefDocument);
         String expectedError = String.format(CrossRefConverter.UNRECOGNIZED_TYPE_MESSAGE, SOME_STRANGE_CROSSREF_TYPE);
-        assertThat(exception.getMessage(), is(equalTo(expectedError)));
+        assertThat(testAppender.getMessages(), CoreMatchers.containsString(expectedError));
     }
 
     @Test
     @DisplayName("An CrossRef document without type throws IllegalArgument exception")
-    public void anCrossRefDocumentWithoutTypeThrowsIllegalArgumentException() throws IllegalArgumentException {
+    public void anCrossRefDocumentWithoutTypeThrowsIllegalArgumentException()
+            throws IllegalArgumentException, InvalidIsbnException,
+            UnsupportedDocumentTypeException, InvalidIssnException {
         CrossRefDocument crossRefDocument = sampleCrossRefDocumentWithBasicMetadata();
         crossRefDocument.setType(null);
-        UnsupportedDocumentTypeException exception =
-                assertThrows(UnsupportedDocumentTypeException.class, () -> toPublication(crossRefDocument));
+        TestAppender testAppender = LogUtils.getTestingAppender(CrossRefConverter.class);
+        toPublication(crossRefDocument);
         String expectedError = String.format(CrossRefConverter.UNRECOGNIZED_TYPE_MESSAGE, "null");
-        assertThat(exception.getMessage(), is(equalTo(expectedError)));
+        assertThat(testAppender.getMessages(), CoreMatchers.containsString(expectedError));
```

Comment:
import static

Reply:
Ups, using the same containsString that was already imported...